low-level/files/opt/seagate/sspl/bin/rabbitmq_setup

```diff
@@ -44,9 +44,9 @@ product=$(sed -nr 's/^product=([^,]+)$/\1/p' $SSPL_CONF | head -1)
 
 # Add required user and vhost to RabbitMQ. Set user permissions etc.
 echo "Creating required RABBITMQ configuration.... "
-python3 $SSPL_BASE_DIR/bin/sspl_rabbitmq_reinit $PRODUCT_NAME || {
+python3 $SSPL_BASE_DIR/low-level/framework/sspl_rabbitmq_reinit $PRODUCT_NAME || {
```

Comment:
same comments, want deployment paths to remain same

Reply:
Done